src/loadYouTubeIframeApi.ts

```diff
@@ -36,23 +46,28 @@ const youtubeAPIReady = () => {
      * if we're on a subsequent call.
      */
     if (youtubeAPIReadyPromise) {
+        log('dotcom', 'loadYT: returning YTAPI promise');
         return youtubeAPIReadyPromise;
     }
     youtubeAPIReadyPromise = new Promise((resolve) => {
+        log('dotcom', 'loadYT: creating YTAPI promise');
         window.onYouTubeIframeAPIReady = () => {
```

Comment:
What if youtube script is ready and calls `window.onYouTubeIframeAPIReady` before this function is defined here?